jekyll/_cci2/sso/setup.adoc

```diff
@@ -0,0 +1,68 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Set up SSO
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to set up SSO for an organization
+:icons: font
+:experimental:
+
+NOTE: SAML Single Sign-On for CircleCI is currently in closed preview.
+
+== Setup SAML SSO
+
+To set up SAML SSO, you must have the _Organization Admin_ role. For informaiton on roles and permissions, see the xref:../roles-and-permissions-overview#[Roles and permissions overview].
+
+=== Access Setup Parameters in CircleCI
+
+To begin setting up a SAML SSO connection in your CircleCI organization, follow these steps:
+
+. In the link:https://app.circleci.com/home/[CircleCI web app], select your organization.
+. Select **Organization Settings** in the sidebar.
+. Select **Single Sign-On (SSO)** from the sidebar.
+. On the SSO Settings page, select **Setup SSO**.
+. Copy the provided **Setup Parameters**. You will need these to configure your Identity Provider (IdP).
+
+=== Configure your Identity Provider (IdP)
+
+. Open your IdP and paste the required configuration details retrieved from CircleCI in the previous section
```

Comment:
For this section it would be great to point ot a few third-party docs pages that show this setup, I wasn't 100% sure from looking at the Okta docs for example. Could you suggest some?

Reply:
Here is an example that we could link to for okta: https://help.okta.com/en-us/content/topics/apps/apps_app_integration_wizard_saml.htm